### PR TITLE
add missing robustness functions in docs

### PR DESCRIPTION
**Pull request by juliettelavoie**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
  - [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] CHANGES.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* The new robustness functions are added to the api doc section.

### CODE REVIEW

**Review by Zeitsperre — APPROVED** [review]

**Comment by Zeitsperre** [comment]
@juliettelavoie Think you could tackle the formatting here?

image

File: xclim/ensembles/_robustness.py [hunk 1]
```diff
@@ -116,8 +118,6 @@ def robustness_fractions(  # noqa: C901
     | Negative change | (cf - cpf)         | 1 - pf - (cf -cpf)     | 1 - pf     |
     +-----------------+--------------------+------------------------+------------+
 
-    Notes
-    -----
     Available statistical tests are :
 
     {tests_doc}
```
Comment (juliettelavoie) [inline xclim/ensembles/_robustness.py:123]: are some tests missing on purpose @aulemahal ? I see ipcc-ar6-c in the example notebook but I can't find it in the code?

**Review by juliettelavoie — COMMENTED** [review]
The top of the robustness section refers to IPCC AR5. Should I change it to AR6 ?

File: xclim/ensembles/_robustness.py (hunk 1 above)
Comment (aulemahal) [reply xclim/ensembles/_robustness.py:123]: Woups, the automatic tests doc generation was done on the wrong method. I think I came up with the mechanism before changing the function name.

**Comment by aulemahal** [comment]
Hmm, indeed, the module's header is a bit outdated.

suggestion:
```rst
This submodule is inspired by and tries to follow the guidelines of the IPCC, more specifically :cite:p:`collins_long-term_2013` (AR5) and :cite:cts:`ipccatlas_ar6wg1` (AR6).
```

**Pull request merged**